### PR TITLE
Swift 4.2 auto-conversion 

### PR DESCRIPTION
**Pull request by sorianog**
PR for #134

### CODE REVIEW

File: ARCL/Source/SceneLocationView.swift [hunk 1]
```diff
@@ -111,7 +111,7 @@ public class SceneLocationView: ARSCNView, ARSCNViewDelegate {
         showsStatistics = false
 
         if showFeaturePoints {
-            debugOptions = [ARSCNDebugOptions.showFeaturePoints]
+            debugOptions = [SCNDebugOptions.showFeaturePoints]
```
Comment (intere) [inline ARCL/Source/SceneLocationView.swift]: Why did you change this from `ARSCNDebugOptions` to `SCNDebugOptions`?

I'm not saying it's wrong, but we're using `ARSCNView` objects (which are more specific than an `SCNView`).

**Comment by sorianog** [comment]
That was what the auto-conversion suggested. I'll change it back to see if
I get an error/warning.
